frontend/e-buddy/src/app/(demo)/bots/[id]/BotDetailPageClient.jsx

```
@@ -338,127 +339,399 @@ export default function BotDetailPageClient({ id, fallbackBot }) {
+                <div className="inline-block mb-3 p-2 bg-orange-100 rounded-full">
+                  <div className="h-12 w-12 rounded-full bg-orange-500 flex items-center justify-center">
+                    <svg
+                      xmlns="http://www.w3.org/2000/svg"
+                      className="h-6 w-6 text-white"
+                      viewBox="0 0 24 24"
+                      fill="none"
+                      stroke="currentColor"
+                      strokeWidth="2"
+                      strokeLinecap="round"
+                      strokeLinejoin="round"
+                    >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)